<launch>
    <node pkg="rosserial_python" type="serial_node.py" name="serial_node">
        <param name="port" value="/dev/ttyACM0"/>
    </node>
    <node pkg="gripper_ctrl" type="vac_ctrl.py" name="vac_ctrl"/>
</launch> 